.github/workflows/test.yaml

```diff
@@ -185,6 +190,14 @@ jobs:
               run: php vendor/bin/simple-phpunit
               working-directory: src/LazyImage
 
+            - name: TwigComponent Dependencies
+              uses: ramsey/composer-install@v2
+              with:
+                working-directory: src/TwigComponent
+                dependency-versions: lowest
+                # needed for php 8.0 to skip symfony/stimulus-bundle
+                composer-options: "--ignore-platform-reqs"
+
```

Comment:
this can all likely be reverted now - the lowest php version and test matrix have all been updated.

Reply:
done